src/index.ts

```diff
@@ -147,7 +184,8 @@ export async function run(): Promise<void> {
     } else {
       core.debug(`Not posting message to slack`);
     }
-  } catch (error) {
+  } catch (error: any) {
```

Comment:
```suggestion
  } catch (error: Error) {
```
?

Reply:
Interestingly, this is the error I got when I did that:

```
Catch clause variable type annotation must be 'any' or 'unknown' if specified.ts(1196)
```

It is news to me...

Reply:
But this explanation made sense to me: https://typescript.tv/best-practices/error-ts1196-catch-clause-variable-type-annotation/


> because the underlying code can throw any type of error, even system generated exceptions, which makes it impossible for TypeScript to know the error type from the start